_pkgdown.yml

```diff
@@ -1,4 +1,71 @@
-url: https://gdrplatform.github.io/gDRutils/
+url: https://gdrplatform.github.io/gDRstyle/
 template:
   bootstrap: 5
 
+reference:
+- title: Assay names
```

Comment:
Perhaps you could add a brief description of the characteristics of the function group?

Reply:
We agree that titles are now quite descriptive.